osbs/api.py

```diff
@@ -602,11 +612,18 @@ def create_worker_build(self, **kwargs):
         for required in ('platform', 'release', 'arrangement_version'):
             if not kwargs.get(required):
                 missing.add(required)
-
         if missing:
             raise ValueError("Worker build missing required parameters: %s" %
                              missing)
 
+        extra = set()
+        for unwanted in ('platforms'):
```

Comment:
well it is prepared for future changes if there would be more unwanted things, and same as in orchestrator